doc/todo/RELEASE

```diff
@@ -25,238 +13,66 @@ Following shell expansions are used in this document:
- cd ~e && git checkout debian
- git merge --no-ff $VERSION -m "Merge for debian release $DVERSION"
-
-UNRELEASED -> unstable
-
- dch --release $DVERSION "New upstream version"
- git add debian/changelog
- git commit -m "Debian Package $DVERSION"
-
-Fix debian/ + document in NEWS what maintainer need to consider, then build the package:
- cp ~elektra-ftp/releases/elektra-$VERSION.tar.gz ~elektra/elektra_$VERSION.orig.tar.gz
- git clean -fdx && gpg -u0B2F0DDC --sign `mktemp` # so that the step afterwards won't abort
- gbp buildpackage -rfakeroot -k0B2F0DDC -sa
-
-Now move and install:
- cd ~elektra && mkdir -p ~elektra/$VERSION/debian/$DVERSION
- mv elektra_$DVERSION* *$DVERSION*.deb elektra_$VERSION.orig.tar.gz ~elektra/$VERSION/debian/$DVERSION/
- sudo dpkg -i ~elektra/$VERSION/debian/$DVERSION/*`dpkg-architecture -qDEB_BUILD_ARCH`.deb
-
-
-Run some basic commands, at least:
- kdb --version | tee ~elektra/$VERSION/version
- kdb qt-gui
-
-Strace what tools are doing:
-
- strace -o ~elektra/$VERSION/mount.strace kdb mount file.ecf user/release_test
- strace -o ~elektra/$VERSION/file.strace kdb file user/release_test/b
- strace -o ~elektra/$VERSION/set.strace kdb set user/release_test/b
- strace -o ~elektra/$VERSION/get.strace kdb get user/release_test/b
- strace -o ~elektra/$VERSION/rm.strace kdb rm user/release_test/b
- strace -o ~elektra/$VERSION/umount.strace kdb umount user/release_test
-
-Run tests:
-
- KDB=kdb kdb run_all -v | tee ~elektra/$VERSION/run_all 2>&1
- KDB=kdb-full kdb-full run_all | tee ~elektra/$VERSION/run_all_full 2>&1
-
-Install libelektra-test of the previous version and run again:
-
- CHECK_VERSION=NO kdb run_all -v > ~elektra/$VERSION/run_all_abi 2>&1
-
-
-== Test external tools ==
-
-
-Check if tools based on Elektra still do:
- dq
-
-Run unit-tests in-source again (because some use installed Elektra):
- make run_all
-
-
-== Test external test suites ==
-
-#https://github.com/oyranos-cms/oyranos/blob/master/src/tests/test2.cpp#L185
-#https://github.com/oyranos-cms/oyranos/blob/master/src/liboyranos_config/oyranos_elektra.c#L195
-
-git clone git@github.com:oyranos-cms/oyranos.git
-cd oyranos
-mkdir build
-cd build
-cmake .. # might fail, simply comment out missing libs
-make -j5 # not needed
-make test2 # or make test-2 for the most of the tests
-src/tests/test2 Elektra
-
-
-== Documentation ==
-
-And the API docu:
- mkdir ~elektra-doc/api/$VERSION/
- cd ~build && cmake -DBUILD_PDF=ON . && rm -rf doc && make html man && cp -ra doc/html doc/latex doc/man ~elektra-doc/api/$VERSION/
-
-Symlink current to latest version and add everything:
- cd ~elektra-doc/api/ && rm current && ln -s $VERSION current && git add current $VERSION && git commit -a -m "$VERSION Release"
-
-
-
+- https://build.libelektra.org/job/libelektra/job/master/ must pass
+- Trigger release pipeline with dry run
+- Inspect the logs from the pipeline
+  - [ ] Error logs (*.error) must be empty
+  - [ ] Check in the (test-*) folders if all tests passed
```

Comment:
```suggestion
  - [ ] Check in the (test-*) folders if the passed tests really were successful
```

I would expect that the build fails if the tests do not pass. The question is if there are some other problems.